bonsai/ethereum-relay/src/uploader/completed_proofs/snark.rs

```diff
@@ -69,40 +70,38 @@ pub(crate) async fn get_snark_receipt(
     Ok(proof)
 }
 
-pub fn tokenize_snark_receipt(proof: &Groth16Seal) -> anyhow::Result<Token> {
-    if proof.b.len() != 2 {
-        anyhow::bail!("hex-strings encoded proof is not well formed");
+pub fn tokenize_snark_receipt(seal: &Groth16Seal) -> anyhow::Result<Token> {
```

Comment:
Are we planning on dropping the `bonsai/ethereum-relay` in favor of the code in `risc0-etheruem`?

Reply:
Yes that is the plan, I just have not removed that on this PR. Should be done by a different one together with some new CI to keep in check the other repo